Mathlib/Analysis/Normed/Group/SemiNormedGrp/Completion.lean

```diff
@@ -93,12 +93,7 @@ instance : Preadditive SemiNormedGrp.{u} where
     rw [NormedAddGroupHom.add_apply, CategoryTheory.comp_apply, CategoryTheory.comp_apply,
       CategoryTheory.comp_apply, @NormedAddGroupHom.add_apply _ _ (_) (_)]
     convert map_add g (f x) (f' x)
-  comp_add _ _ _ _ _ _ := by
-    ext
-    -- Porting note: failing simps probably due to instance synthesis issues with concrete
-    -- cats; see the gymnastics below for what used to be
-    rw [NormedAddGroupHom.add_apply, CategoryTheory.comp_apply, CategoryTheory.comp_apply,
-      CategoryTheory.comp_apply, @NormedAddGroupHom.add_apply _ _ (_) (_)]
+  comp_add _ _ _ _ _ _ := by rfl
```

Comment:
Can another reviewer double-check this is fine?

Reply:
This is not my expertise but I'm indeed a bit suspicious about the fact that `add_comp` can't be `rfl` but `comp_add` can.

If `rfl` is okay, then we don't need the `by`:
```suggestion
  comp_add _ _ _ _ _ _ := rfl
```

Reply:
this part has been removed completely in the meantime